VALUES.md

```diff
@@ -0,0 +1,66 @@
+# Ayo.js Community Values
+
+Ayo.js is about the humans that help make it happen. It's a project aimed at
+creating a new foundation of project governance and management that brings
+**humans** and their concerns front and center.
+
+You can pronounce it however you want: be it “ey-yo”, “I-oh”, or “Awooooo” — the
+Ayo.js project strives to create an environment where you can feel at home and
+focus on your contributions to this nice shared space that we all benefit from.
+
+## Ayo.js is more than the code in its repository.
+
+Documentation, long-term support, translations and localizations,
+package management, libuv, V8 and Chakra, builds/ops, testing and its
+infrastructure, hardware and IoT, Android, streams, community events,
+facilitation, moderation and education. These and more are all concerns that
+can be handled separately and by members who might be much better qualified
+for these individual tasks than a central decision body.
+
+Ayo.js wants to go further than Node.js did in spreading responsibility across
+the project, so everyone can focus and specialize better than before.
+Contributors should be able to organize and manage their own subprojects
+in a way that is effective and efficient for their unique needs.
+
+## Humans are more important than red tape.
+
+If governance policy is not serving the values set forth here, that governance
+policy should change with the required agility. Following the governance policy
+to the letter should not take precedence over staying true to our stated values,
+not to mention the well-being of our humans.
+
+## Humans are more important than APIs and Benchmarks.
+
+Technical discussions are difficult, especially in large communities with
+members with all sorts of perspectives. You are bound to find yourself in
+situations with no clear consensus, with new concerns being brought up
+on a regular basis, and things that might seem insignificant in other
+contexts suddenly exploding.
+
+The Ayo.js project's perspective is that when this happens, it often takes a
+toll on the well-being of the members participating, or it prevents potential
+members from contributing again, if at all. It also recognizes that the
+difference between individual choices is less important than the ability to
+come to a perfect technical decision.
+
+Projects change, evolve, and find themselves dealing with new concerns all the
+time. In all technical matters, Ayo.js will take into account that in the end,
+it is the people making those decisions that matter most, not the decisions
+themselves. It’s ok for code to be imperfect.
+
+## Humans are more important than corporations.
+
+All sorts of corporations and other organizations around the world use and
+benefit from the software underlying Ayo.js. Many contributors, in fact, might
+have their efforts in the project directly funded by them. But this can often
```

Comment:
This is a complete shot in the dark and I don't expect it'll be taken at face value, but for me at least, some of the impact of this paragraph is lost with the current writing. 

"While potentially adding a significant amount of additional resources to the project, these contributions can precipitate into such direct reliance on institutions that contribute monetary funds to the project that the potential for abuse of their power or outright attempts to usurp control of a project can leave individual contributors or smaller institutions wondering if they can ever have any influence over the project at all"